core/client/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -56,6 +56,13 @@ public long getCapacityBytes() {
   }
 
   /**
+   * @return the available bytes of the worker
+   */
+  public long getAvailableBytes() {
```

Comment:
I use the alluxio.user.block.size.bytes.default (512M) and add a new property alluxio.user.file.write.capacity.reserved.ratio(default 10), the reserved capacity is 5G. The default value of ratio is suitable?